ui/ducks/send/send.test.js

```diff
@@ -1410,6 +1410,7 @@ describe('Send Slice', () => {
             type: 'send/updateGasFees',
             payload: {
               gasPrice: '0x0',
+              wasManuallyEdited: true,
```

Comment:
Oops, changed one too many:
```suggestion
              manuallyEdited: true,
```